metasv/process_age_alignment.py

```diff
@@ -392,9 +310,18 @@ def process_age_records(age_records, sv_type="INS", ins_min_unaligned=10, min_in
     elif sv_type == "INV":
         pass
 
+    # Add some features to an info dict
+    info = defaultdict(float)
+    info["BA_NUM_GOOD_REC"] = len(good_age_records) if good_age_records else 0
+    for rec in good_age_records:
+        info["BA_FLANK_PERCENT"] = max(info["BA_FLANK_PERCENT"], rec.flank_percent)
+        info["BA_NFRAGS"] = max(info["BA_NFRAGS"], rec.nfrags)
+        info["BA_NUM_ALT"] = max(info["BA_NUM_ALT"], rec.n_alt)
+        info["BA_PERCENT_MATCH"] = max(info["BA_PERCENT_MATCH"], rec.percent)
```

Comment:
These are all reasonable. In fact, Alex used all of these in his script to process AGE alignments.